docs/docs/toolkits.md

```diff
@@ -0,0 +1,198 @@
+# Toolkits
+
+This page contains information about building and using toolkits in Goose. Toolkits are a way to extend Goose's capabilities by adding new tools and functionalities. You can create your own toolkits or use the existing ones provided by Goose.
+
+## Using Toolkits
+
+### Toolkits defined in Goose
+
+Using Goose with toolkits is simple. You can add toolkits to your profile in the `profiles.yaml` file. Here's an example of how to add `my-toolkit` toolkit to your profile:
+
+```yaml
+my-profile:
+  provider: openai
+  processor: gpt-4o
+  accelerator: gpt-4o-mini
+  moderator: passive
+  toolkits:
+    - my-toolkit
+```
+
+Then run Goose with the specified profile:
+
+```sh
+goose session start --profile my-profile
+```
+
+### Toolkits defined in Goose Plugins
+
+1. First make sure that `goose-plugins` is intalled with Goose:
+```sh
+pipx install goose-ai --preinstall goose-plugins
```

Comment:
do we want to suggest this or just `pipx intsall goose-plugins --include-deps`?

Reply:
i have no idea which is the correct one haha

Reply:
is that the one we decided to tell everyone?